libcalico-go/lib/backend/model/keys.go

```diff
@@ -23,13 +23,19 @@ import (
 	"time"
 
 	log "github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 
+	apiv3 "github.com/projectcalico/api/pkg/apis/projectcalico/v3"
 	"github.com/projectcalico/calico/libcalico-go/lib/json"
 	"github.com/projectcalico/calico/libcalico-go/lib/namespace"
 	"github.com/projectcalico/calico/libcalico-go/lib/net"
 )
 
+const (
+	metadataAnnotation = "projectcalico.org/metadata"
```

Comment:
We have this defined in a few places now - should we put it in a common location instead?